lib/utility.js

```diff
@@ -81,7 +81,7 @@ export async function manFiles() {
 		}
 	}
 
-	return [ 'docs/core/man/*.[[:digit:]].md' ]
+	return [ join('docs',component,'*.[[:digit:]].md') ]
```

Comment:
Again, this pigehonhole stuff is adding complexity to build-scripts and the like, while not providing any value to a user.

A user cares about "sieve commands".  It doesn't care that these are "pigeonhole sieve commands".  In fact, this is more confusing: a user now has to learn about the difference between pigeonhole and dovecot to understand where to find the documentation.  And what is that difference?  Does a user care?

Reply:
Our own build system cares though as dovecot and pigeonhole are separate. I need some way to ship manpages by component.